src/languageservice/serviceclient.ts

```diff
@@ -249,11 +254,19 @@ export default class SqlToolsServiceClient {
         let client = new LanguageClient(Constants.sqlToolsServiceName, serverOptions, clientOptions);
         client.onReady().then( () => {
             this.checkServiceCompatibility();
+
         });
+        client.onNotification(LanguageServiceContracts.DefinitionSentNotification.type, this.handleLanguageServiceDefinitionSentNotification());
 
         return client;
     }
 
+    private handleLanguageServiceDefinitionSentNotification(): NotificationHandler<LanguageServiceContracts.DefinitionSentParams> {
+        return (event: LanguageServiceContracts.DefinitionSentParams): void => {
+            Telemetry.sendTelemetryEvent('DefinitionRequested');
```

Comment:
the volume of this event will likely be pretty high.  is there a reason we need to capture a separate event for each definition request vs. an aggregated metric?